clients/scheduler/src/utils/dates/calendar.ts

```diff
@@ -128,9 +128,10 @@ export const generateDays = (date: Date) => {
         dateString: getDateString(year, month, i),
       })
     } else {
+      const day = new Date(year, month, i)
       days.push({
         date: i,
-        disabled: false,
+        disabled: (maxDate && maxDate < day) || disabledDates.has(day.toLocaleDateString()) ? false: true,
```

Comment:
This makes me more confident in my comment above, we are setting `disabled` to `false` when the `disabledDates` set has the day.